crates/turbopack-tests/tests/snapshot/css/absolute-uri-import/input/index.css

```diff
@@ -0,0 +1,7 @@
+@import "https://example.com/stylesheet1.css";
+@import url("https://example.com/stylesheet2.css");
+@import "./other.css";
```

Comment:
Swap these two lines to verify that external imports are correctly hoisted over normal imports